Streetcode/Streetcode.DAL/Repositories/Realizations/Base/RepositoryWrapper.cs

```diff
@@ -135,6 +137,19 @@ public IPartnersRepository PartnersRepository
         }
     }
 
+    public IRelatedFigureRepository RelatedFigureRepository
+    {
+        get
+        {
+            if (_relatedFigureRepository == null)
+            {
+                _relatedFigureRepository = new RelatedFigureRepository(_streetcodeDbContext);
+            }
```

Comment:
Please replace `__value == null__` with `__value is null__` in all scoped accesors, which is considered a best practice since .NET 5.